recipe_scrapers/wholefoods.py

```diff
@@ -0,0 +1,7 @@
+from ._abstract import AbstractScraper
+
+
+class WholeFoods(AbstractScraper):
+    @classmethod
+    def host(self, domain="com"):
+        return f"www.wholefoodsmarket.{domain}"
```

Comment:
I'm not sure what we want to do for coding style. On one hand this makes it very simple, on the other there is some magic without the other functions explicitly defined as using the schema parser.

Reply:
I wasn't sure if there was a pattern for parsers that use the standard schema.